main.go

```diff
@@ -220,7 +220,7 @@ func run(args []string) error {
 	// Set up file watchers (if requested)
 	watcher := make(chan bool, 1)
 	if *timedReload > 0 {
-		go watchFiles([]string{*keystorePath, *caBundlePath}, *timedReload, watcher)
+		go watchFiles([]string{*keystorePath}, *timedReload, watcher)
```

Comment:
Nope. It doesn't actually work to reload the CA bundle, ever since we rewrote the cert reloading logic a few months back. That's ok though, the CA bundle should rarely if ever change at runtime (whereas leaf certificates can/should be short-lived).